src/bussinessLogic/ManageData.java

```diff
@@ -0,0 +1,40 @@
+package bussinessLogic;
+
+import interfaces.ArrayManager;
+import interfaces.DataManager;
+import interfaces.MapManager;
+
+public class ManageData implements DataManager {
+    private final MapManager manageMap;
+    private final ArrayManager manageArray;
+    // int for calculating how many times a letter is displayed in the evaluation
+    // sequence
+    private int alphabetMultiplier = 1;
```

Comment:
I have seen that there are multiple constants in several classes inside this project.
An idea would be to create a new class inside which these constants could be moved and then, referenced throughout the project.
This way there will be _one single point of truth_ regarding those.

At the moment one of the constants is duplicated. See **ManageMap** class and **numberOfLetters** property together with **ManageArray** class and **numberOfLetters** property.

Reply:
> An idea would be to create a new class inside which these constants could be moved and then, referenced throughout the project.
> This way there will be one single point of truth regarding those.

That's exactly my thought also. This is the way I was planning to solve the initialization problem on the CBM program :grinning: